std_msgs/Header header
uint8[] descriptor
uint64 descriptor_size
uint8[] imgBuffer
uint64 imgSize